driver/src/main/java/org/neo4j/driver/internal/async/NetworkConnection.java

```diff
@@ -194,6 +184,25 @@ public BoltProtocol protocol() {
         return protocol;
     }
 
+    @Override
+    public void bindTransaction(UnmanagedTransaction transaction) {
```

Comment:
Might worth cover it with tests. The code is quite simple, but it's good avoid any future change break some desired behaviour by accident.